vehicle/mkvi/software/sensing/BUILD

```diff
@@ -0,0 +1,32 @@
+load("//bazel/tools:defs.bzl", "can_api_files")
+load("//bazel:defs.bzl", "cc_firmware")
+
+exports_files([
+    "sensing.yml",
+])
+
+cc_firmware(
+    name = "sensing",
+    srcs = [
+        "sensing.c",
+        "sensing.h",
+    ],
+    btldr = "//projects/btldr:brakelight_bspd_btldr",
```

Comment:
Maybe a question for @jack-greenberg but is there a separate btldr config for the sensing board?

Reply:
There will be! @richardli03 please comment out everything relating to the bootloader. I will create a PR adding the bootloader to all MKVI code.